headers/fmi3LsBusFlexRay.h

```diff
@@ -0,0 +1,542 @@
+#ifndef fmi3LsBusFlexRay_h
+#define fmi3LsBusFlexRay_h
+
+/*
+This header file declares FlexRay bus specific constants and data types as defined by the
+fmi-ls-bus layered standard specification (https://github.com/modelica/fmi-ls-bus).
+
+It should be used when creating FlexRay Network FMUs according to the fmi-ls-bus layered standard.
```

Comment:
"fmi-ls-bus layered standard" is written in capital in some other header files.

Reply:
I addressed it via issue https://github.com/modelica/fmi-ls-bus/issues/200 
Currently (not only) in the FlexRay part is a kind of mixture. I think the best way is to decide how we want to write it (I'm fine with both options) and then I will fix it within the complete document, header files, Impl. Guide and the landing pages.